plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -163,4 +163,27 @@
       </processFactory>
    </extension>
 
+   <extension
+         point="org.eclipse.core.runtime.preferences">
+      <initializer
+            class="com.google.cloud.tools.eclipse.appengine.localserver.PreferencesInitializer">
+      </initializer>
+   </extension>
+   <extension
+         point="org.eclipse.ui.preferencePages">
+      <page
+            category="com.google.cloud.tools.eclipse.preferences.main"
+            class="com.google.cloud.tools.eclipse.preferences.areas.AreaBasedPreferencePage"
+            id="com.google.cloud.tools.eclipse.preferences.appengine.localserver"
+            name="Run/Debug">
```

Comment:
Looking at it again, it looks strange.  I think I'll keep it as _Run/Debug_, to mimic the existing Eclipse _Run/Debug_ pref page (i.e., things to do with launching), and put _App Engine_ into the area title instead.